src/page-module/files/__name@dasherize__/__name@dasherize__.component.ts

```diff
@@ -1,4 +1,6 @@
-import { Component<% if (isNgrxInstalled) { %>, ChangeDetectionStrategy<% } %> } from '@angular/core';
+import { Component<% if (isNgrxInstalled) { %>, ChangeDetectionStrategy<% } %> } from '@angular/core';<% if (facade) { %>
+import { <%= classify(section + ' ' + ((hasParent) ? (parent + ' ') : '') + name) %>PageFacade } from './<%= dasherize(name) %>.facade';
```

Comment:
@Goodmain same as above, this `classify` call can be shortened.

Reply:
@dmitryusenko Fixed